update_compilers/install_libraries.sh

```diff
@@ -16,6 +16,14 @@ else
     git -C libs/kvasir/mpl/trunk pull origin development
 fi
 
+# add Brigand metaprogramming library
+if [ ! -d "libs/brigand/trunk" ]; then
```

Comment:
If you take a look at the other libs, they define a function here so we can download selected github releases (As mentioned in the other PR) It would be a good idea if this had the same behaviour. Feel free to use one of the other functions as a template